storage/tests/c/flash.h

```diff
@@ -24,19 +24,21 @@
 #include <stdlib.h>
 #include "secbool.h"
 
-#define FLASH_SECTOR_COUNT 24
+#define FLASH_BYTE_ACCESS 1
 
-secbool __wur flash_unlock_write(void);
-secbool __wur flash_lock_write(void);
+#include "flash_common.h"
+#include "test_layout.h"
 
-const void *flash_get_address(uint8_t sector, uint32_t offset, uint32_t size);
+uint32_t flash_sector_size(uint16_t sector);
 
-secbool __wur flash_erase_sectors(const uint8_t *sectors, int len,
-                                  void (*progress)(int pos, int len));
-static inline secbool flash_erase(uint8_t sector) {
-  return flash_erase_sectors(&sector, 1, NULL);
-}
-secbool __wur flash_write_byte(uint8_t sector, uint32_t offset, uint8_t data);
-secbool __wur flash_write_word(uint8_t sector, uint32_t offset, uint32_t data);
+secbool flash_area_erase_bulk(const flash_area_t *area, int count,
+                              void (*progress)(int pos, int len));
+
+secbool flash_write_byte(uint16_t sector, uint32_t offset, uint8_t data);
+
+secbool flash_write_word(uint16_t sector, uint32_t offset, uint32_t data);
+
+secbool flash_area_write_byte(const flash_area_t *area, uint32_t offset,
+                              uint8_t data);
```

Comment:
Please remove `flash_area_write_byte()`. Already declared in `flash_common.h`.

Reply:
https://github.com/trezor/trezor-firmware/commit/626e6d0954357a79c03f7e8f80a947f9414f78f7